### PR TITLE
fixed react key warning

### PR DESCRIPTION
**Pull request by yayitswei**
Fixes the warning about a non-scalar react key causing spurious refreshes.

### CODE REVIEW

File: src/app/client.cljs [hunk 1]
```diff
@@ -128,7 +128,7 @@
                                (comp/transact! this [(add-new-item {:item/id (random-uuid)})]))}
                    (dom/i :.plus.icon)))))))
 
-(def ui-item-list (comp/factory ItemList {:keyfn :item-list/all-items}))
+(def ui-item-list (comp/factory ItemList {:keyfn (constantly ::ui-item-list)}))
```
Comment (awkay) [inline src/app/client.cljs:131]: this should not be...why this change?  The arg is a function that returns a key...if anything none is needed.  This particular change is actually quite bad, as it would cause the react key to change constantly.

**Comment by awkay** [comment]
So, I've been considering this, and I think my policy on these video files is that they should not be changed.  The videos were made with a certain version of Fulcro, in order to get across core concepts that are not changing.  They are not meant to be templates of any time, simply throw-away code that demonstrates concepts. As such, I think it does not make sense for them to change over time without re-recording videos.  So, I think PRs on this repo will generally not be accepted.

Thanks for trying to help.  If this were the `fulcro-template` I'd be very glad for any perfection fixes, but I consider these part of a record of a *performance* not of an idealized example.

**Comment by yayitswei** [comment]
Thanks for taking the time to explain your reasoning on this, makes sense.
